src/visual/aria.ts

```diff
@@ -202,7 +202,11 @@ export default function ariaVisual(ecModel: GlobalModel, api: ExtensionAPI) {
                         seriesType: getSeriesTypeName(seriesModel.subType as SeriesTypes)
                     });
 
-                    const data = seriesModel.getData();
+                    let data = seriesModel.getData();
+                    const columnsToExclude = labelModel.get(['data', 'columnsToExclude']);
+                    if (columnsToExclude) {
+                        data = data.filterSelf(idx => !columnsToExclude?.includes(idx));
```

Comment:
This will make the elements in the chart not rendered. What we want is only to change the aria-label instead of the original chart.

Reply:
I wanted to do the filtering before the check with `maxDataCnt` - but of course using `filterSelf` was all wrong.
I have switched back to the old solution in commit [77580e9](https://github.com/apache/echarts/pull/20038/commits/77580e930fa5083752e0d632e4f6f1d9631dabac).
Regarding `maxDataCnt`, I guess the length of the array `columnsToExclude` could be added to  `maxDataCnt` to compensate for the excluded columns:
```
if (columnsToExclude) {
        maxDataCnt += columnsToExclude.length;
}
```
What do you think?

Reply:
What kind of filtering does this PR want to provide? Do you want to hiding some bars or only eliminate some dimensions from aria-label (in which case you shouldn't call `filterSelf` because it changes the data itself)?

Reply:
> What kind of filtering does this PR want to provide? Do you want to hiding some bars or only eliminate some dimensions from aria-label (in which case you shouldn't call `filterSelf` because it changes the data itself)?

Only eliminate some dimensions form aria-label. So `filterSelf` was a [mistake ](https://github.com/apache/echarts/pull/20038#discussion_r1647911321).